README.md

```diff
@@ -16,7 +16,7 @@ jsEnv := new org.scalajs.jsenv.selenium.SeleniumJSEnv(BROWSER)
 jsEnv in Test := new org.scalajs.jsenv.selenium.SeleniumJSEnv(BROWSER)
 ```
 where the `BROWSER` can be either `org.scalajs.jsenv.selenium.Firefox` or
-`org.scalajs.jsenv.selenium.Chrome`.
+`new org.scalajs.jsenv.selenium.Chrome`.
```

Comment:
These should actually become `org.scalajs.jsenv.selenium.Firefox()` and `org.scalajs.jsenv.selenium.Chrome()`.